client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/taxi_gateway/ITaxiGateway.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.data.remote.gateway.taxi_gateway
```

Comment:
the gateway interface must be in domain layer and its implementation must be in data layer you broke the architecture